app.js

```diff
@@ -270,25 +270,25 @@ var myChart = new Chart(ctx, barChart);
     
 }
 
-new Chart(document.getElementById("doughnut-chart"), {
-    type: 'pie',
-    data: {
-      labels: prodNames,
-      datasets: [
-        {
-          label: "Saved Data",
-          backgroundColor: ['red', 'aqua', 'aquamarine', 'blue', 'blueviolet', 'brown', 'cadetblue', 'chartreuse', 'darkblue', 'deeppink', 'gold', 'green', 'yellow','red', 'aqua', 'aquamarine', 'blue', 'blueviolet', 'brown', 'cadetblue'],
-          data: prodLikes
-        }
-      ]
-    },
-    options: {
-      title: {
-        display: true,
-        text: 'Saved Data From LocalStorage'
-      }
-    }
-});
+// new Chart(document.getElementById("doughnut-chart"), {
+//     type: 'pie',
+//     data: {
+//       labels: prodNames,
+//       datasets: [
+//         {
+//           label: "Saved Data",
+//           backgroundColor: ['red', 'aqua', 'aquamarine', 'blue', 'blueviolet', 'brown', 'cadetblue', 'chartreuse', 'darkblue', 'deeppink', 'gold', 'green', 'yellow','red', 'aqua', 'aquamarine', 'blue', 'blueviolet', 'brown', 'cadetblue'],
+//           data: prodLikes
+//         }
```

Comment:
the chart looks good and the data is being saved to local storage however there is still a bug in the randomizer where every once in a few clicks one image will repeat in the same position.